main/acle.md

```diff
@@ -4932,6 +5044,1381 @@ assertions, which may speed up access to aligned data (and will fault
+  void svmopa_za64[_f64](uint64_t tile, svbool_t pn, svbool_t pm,
+                         svfloat64_t zn, svfloat64_t zm);
+```
+
+#### BFMOPS, FMOPS (widening), SMOPS, UMOPS
+
+``` c
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za32[_bf16](uint64_t tile, svbool_t pn, svbool_t pm,
+                          svbfloat16_t zn, svbfloat16_t zm);
+
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za32[_f16](uint64_t tile, svbool_t pn, svbool_t pm,
+                         svfloat16_t zn, svfloat16_t zm);
+
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za32[_s8](uint64_t tile, svbool_t pn, svbool_t pm,
+                        svint8_t zn, svint8_t zm);
+
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za32[_u8](uint64_t tile, svbool_t pn, svbool_t pm,
+                        svuint8_t zn, svuint8_t zm);
+
+  // Only if __ARM_FEATURE_SME_I16I64 != 0
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za64[_s16](uint64_t tile, svbool_t pn, svbool_t pm,
+                         svint16_t zn, svint16_t zm);
+
+  // Only if __ARM_FEATURE_SME_I16I64 != 0
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za64[_u16](uint64_t tile, svbool_t pn, svbool_t pm,
+                         svuint16_t zn, svuint16_t zm);
+```
+
+#### FMOPS (non-widening)
+
+``` c
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za32[_f32](uint64_t tile, svbool_t pn, svbool_t pm,
+                         svfloat32_t zn, svfloat32_t zm);
+
+  // Only if __ARM_FEATURE_SME_F64F64 != 0
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmops_za64[_f64](uint64_t tile, svbool_t pn, svbool_t pm,
+                         svfloat64_t zn, svfloat64_t zm);
+```
+
+#### RDSVL
+
+The following intrinsics read the length of a streaming vector:
+
+``` c
+  // Return the number of bytes in a streaming vector.
+  // Equivalent to svcntb() when called in streaming mode.
+  __attribute__((arm_streaming_compatible, arm_preserves_za))
+  uint64_t svcntsb();
+
+  // Return the number of halfwords in a streaming vector.
+  // Equivalent to svcnth() when called in streaming mode.
+  __attribute__((arm_streaming_compatible, arm_preserves_za))
+  uint64_t svcntsh();
+
+  // Return the number of words in a streaming vector.
+  // Equivalent to svcntw() when called in streaming mode.
+  __attribute__((arm_streaming_compatible, arm_preserves_za))
+  uint64_t svcntsw();
+
+  // Return the number of doublewords in a streaming vector.
+  // Equivalent to svcntd() when called in streaming mode.
+  __attribute__((arm_streaming_compatible, arm_preserves_za))
+  uint64_t svcntsd();
+```
+
+`svcntsb()` is equivalent to an RDSVL instruction with an immediate
+operand of 1. There are no intrinsics that map directly to other
+immediate operands, or to the ADDSVL and ADDSPL instructions, but it is
+possible to write these operations using normal C arithmetic. For example:
```

Comment:
```suggestion
`svcntsb()` is equivalent to an `RDSVL` instruction with an immediate
operand of 1. There are no intrinsics that map directly to other
immediate operands, or to the `ADDSVL` and `ADDSPL` instructions, but it is
possible to write these operations using normal C arithmetic. For example:
```

Reply:
Here too I think the Arm ARM standardises on not using fixed-width font for instruction names (outside of actual code, that is).  Would it be OK to do the same here?